astropy/stats/funcs.py

```diff
@@ -1149,11 +1153,9 @@ def func(s):
 
 def _mpmath_kraft_burrows_nousek(N, B, CL):
     '''Upper limit on a poisson count rate
-
     The implementation is based on Kraft, Burrows and Nousek in
     `ApJ 374, 344 (1991) <http://adsabs.harvard.edu/abs/1991ApJ...374..344K>`_.
     The XMM-Newton upper limit server used the same formalism.
-
```

Comment:
Minor nit since this is a private function, but why did you remove this blank lines?  There should be blanks lines before each section.